src/bo4e/bo/steuerbareressource.py

```diff
@@ -0,0 +1,40 @@
+"""
+Contains steuerbare Ressource class
+and corresponding marshmallow schema for de-/serialization
+"""
+
+from typing import Annotated, Optional
+
+from pydantic import Field
+
+from ..com.konfigurationsprodukt import Konfigurationsprodukt
+from ..enum.marktrolle import Marktrolle
+from ..enum.steuerkanalsleistungsbeschreibung import SteuerkanalsLeistungsbeschreibung
+from ..enum.typ import Typ
+from ..utils import postprocess_docstring
+from .geschaeftsobjekt import Geschaeftsobjekt
+
+# pylint: disable=too-many-instance-attributes, too-few-public-methods
+
+
+@postprocess_docstring
+class SteuerbareRessource(Geschaeftsobjekt):
+    """
+    Object containing information about a steuerbare Ressource
+
+    .. raw:: html
+
+        <object data="../_static/images/bo4e/bo/SteuerbareRessource.svg" type="image/svg+xml"></object>
+
+    .. HINT::
+        `Messlokation JSON Schema <https://json-schema.app/view/%23?url=https://raw.githubusercontent.com/BO4E/BO4E-Schemas/{__gh_version__}/src/bo4e_schemas/bo/SteuerbareRessource.json>`_
+
+    """
+
+    typ: Annotated[Optional[Typ], Field(alias="_typ")] = Typ.STEUERBARERESSOURCE
+
+    id: Optional[str] = None
```

Comment:
This should not be just id, but steuerbare_ressource_id (see dotnet lib), also please note that we do not use camelCase here but snake_case